### PR TITLE
Universe with actionable proposals component

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

The “Actionable Proposals” page should display all actionable proposals grouped by universe (Nns/Sns). Here, we add a wrapper component that will represent a universe and provide a slot for rendering universe proposals. Proposals can’t be provided via props because `Nns` and `Sns` have different types.

# Changes

- Add a new component.

# Tests

- Add PO for the new component.
- Add tests for UniverseWithActionableProposals.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

File: frontend/src/lib/components/proposals/UniverseWithActionableProposals.svelte [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  export let universe: Universe;
+</script>
+
+<div class="container" data-tid="universe-with-actionable-proposals-component">
+  <div class="summary" data-tid="projects-summary">
+    <h1 class="title">
+      <span class="title">{universe.title}</span>
+
+      <UniverseLogo {universe} framed />
```
Comment (dskloetd) [inline frontend/src/lib/components/proposals/UniverseWithActionableProposals.svelte:14]: Would it make sense to reuse [UniversePageSummary](https://github.com/dfinity/nns-dapp/blob/5b2569bf757014e5b43bdd514be2d8fdcd87f9c0/frontend/src/lib/components/universe/UniversePageSummary.svelte) here?

File: frontend/src/tests/page-objects/UniverseWithActionableProposals.page-object.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    ).map((el) => new UniverseWithActionableProposalsPo(el));
+  }
+
+  async getTitle(): Promise<string> {
+    return (
+      await assertNonNullish(this.root.querySelector("h1")).getText()
+    ).trim();
+  }
+
+  getProposalCardPos(): Promise<ProposalCardPo[]> {
```
Comment (dskloetd) [inline frontend/src/tests/page-objects/UniverseWithActionableProposals.page-object.ts:29]: The cards aren't rendered by `UniverseWithActionableProposals` so I don't think the getter belongs here.
I would put it in the PO of the component that actually renders the cards in the slot.

File: frontend/src/lib/components/proposals/UniverseWithActionableProposals.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/proposals/UniverseWithActionableProposals.svelte:14]: Good suggestion. Let’s try it. The styles look similar to the ones used on the proposals page.

File: frontend/src/tests/page-objects/UniverseWithActionableProposals.page-object.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/tests/page-objects/UniverseWithActionableProposals.page-object.ts:29]: Removed 👍

File: frontend/src/tests/page-objects/UniverseWithActionableProposals.page-object.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    element: PageObjectElement
+  ): Promise<UniverseWithActionableProposalsPo[]> {
+    return Array.from(
+      await element.allByTestId(UniverseWithActionableProposalsPo.TID)
+    ).map((el) => new UniverseWithActionableProposalsPo(el));
+  }
+
+  async getTitle(): Promise<string> {
+    return (
+      await assertNonNullish(this.root.querySelector("h1")).getText()
```
Comment (dskloetd) [inline frontend/src/tests/page-objects/UniverseWithActionableProposals.page-object.ts:24]: Please use [UniversePageSummaryPo](https://github.com/dfinity/nns-dapp/blob/5b2569bf757014e5b43bdd514be2d8fdcd87f9c0/frontend/src/tests/page-objects/UniversePageSummary.page-object.ts).

Comment (mstrasinskis) [reply frontend/src/tests/page-objects/UniverseWithActionableProposals.page-object.ts]: Done.

**Review by dskloetd — APPROVED** [review]

**Pull request merged**